pkg/apis/core/validation/validation_test.go

```diff
@@ -26715,3 +26715,59 @@ func TestValidatePodResize(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateNodeSwapStatus(t *testing.T) {
+	var nilSwapCapacity *int64 = nil
```

Comment:
this var is pointless - can we just say `nil` in the testcase?

Reply:
sure. removed.